src/wiring_diagrams/WiringDiagrams.jl

```diff
@@ -8,16 +8,19 @@ include("MonoidalDirected.jl")
 include("MonoidalUndirected.jl")
 include("Algorithms.jl")
 include("Expressions.jl")
-include("Serialization.jl")
-include("GraphML.jl")
-include("JSON.jl")
+include("ScheduleUndirected.jl")
```

Comment:
File name and module name don't match see earlier comment about shortening the module name.